Quicksort_3_way/Quicksort_3_way.c

```diff
@@ -0,0 +1,102 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <time.h>
+typedef void qsort3way_swap(void *a, void *b);
+typedef int qsort3way_cmp(void const *a, void const *b);
+static void qsort3way_aux(char *array_begin, char *array_end, size_t size,
+                              qsort3way_cmp *cmp, qsort3way_swap *swap) 
+ {
+    if (array_begin < array_end)
+    {
+     char *i = array_begin + size;
+     char *lower = array_begin;
+     char *greater = array_end;
+     // 3-way partition based quick sort 
```

Comment:
you didn't keep four space indentation

Reply:
@asha15 IT WAS FOUR SPACE IDENTATION BEFORE I DID AS YOU TOLD ME TO AND THEY CHANGED

Reply:
i asked you to do it for the sub parts

Reply:
i made the changes @asha15